File: test/fixtures/output/csharp/httpclient/custom-method.cs
var client = new HttpClient();
var request = new HttpRequestMessage
{
    Method = new HttpMethod("PROPFIND"),
    RequestUri = new Uri("https://httpbin.org/anything"),
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}

File: src/targets/swift/urlsession/fixtures/custom-method.swift
import Foundation

let url = URL(string: "https://httpbin.org/anything")!
var request = URLRequest(url: url)
request.httpMethod = "PROPFIND"
request.timeoutInterval = 10

let (data, response) = try await URLSession.shared.data(for: request)
print(String(decoding: data, as: UTF8.self))